projectmanagement/src/main/java/life/qbic/projectmanagement/project/ProjectTitle.java

```diff
@@ -15,4 +16,11 @@ public record ProjectTitle(String title) {
     }
   }
 
+  public static ProjectTitle from(String s) {
+    return new ProjectTitle(s);
+  }
+
+  public String get() {
+    return title();
+  }
```

Comment:
Why do we need this method?

Reply:
Yes please. Alternatively you can rename the component to `value` instead.
Having two getters for one field is not ideal thou.